Js.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function(){
+
+//HAD TO TIME BOX THIS ONE, WAS GETTING ERRORS ON MY RETURN STATEMENT, I THINK MY ORDER IS WAY OFF HERE, GONNA TACKLE THE NEXT ASSIGNMENT AND COME BACK TO IT TIME WILLING.
+
+//set a variable time, then write function to make it countdown.
+  var time = 0;
+  var timer= setInterval(timer, 1000);
```

Comment:
Alright I'll give it ago. Put it down to work on portfolio page for bit. :)
Thanks
On Feb 28, 2016 21:54, "Robert Thomas" <notifications@github.com> wrote:

> In Js.js
> <https://github.com/ga-wdi-exercises/final-countdown/pull/46#discussion_r54365052>
> :
>
> > @@ -0,0 +1,26 @@
> > +$(document).ready(function(){
> > +
> > +//HAD TO TIME BOX THIS ONE, WAS GETTING ERRORS ON MY RETURN STATEMENT, I THINK MY ORDER IS WAY OFF HERE, GONNA TACKLE THE NEXT ASSIGNMENT AND COME BACK TO IT TIME WILLING.
> > +
> > +//set a variable time, then write function to make it countdown.
> > +  var time = 0;
> > +  var timer= setInterval(timer, 1000);
>
> I don't think you can have a setInterval call itself, as you're doing
> here. Usually it looks something like this:
>
> setInterval(oneSec, 1000);function oneSec(){
>   alert("A second passed!");
> }
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-exercises/final-countdown/pull/46/files#r54365052>
> .
>